src/components/templates/ConversationSpace/ConversationSpace.tsx

```diff
@@ -18,19 +21,38 @@ import TableHeader from "components/molecules/TableHeader";
 import { UserList } from "components/molecules/UserList";
 import { TablesUserList } from "components/molecules/TablesUserList";
 
+import { BackButton } from "components/atoms/BackButton";
+
 import { TABLES } from "./constants";
 
 import "./ConversationSpace.scss";
 
+export type ConversationSpaceProps = {
+  venue?: WithId<AnyVenue>;
+};
+
 // @debt refactor this to pass in venue as a prop
-export const ConversationSpace: React.FunctionComponent = () => {
-  const venue = useSelector(currentVenueSelectorData);
+export const ConversationSpace: React.FunctionComponent<ConversationSpaceProps> = ({
+  venue,
+}) => {
+  const { parentVenue } = useRelatedVenues({
+    currentVenueId: venue?.id,
+  });
+  const parentVenueId = parentVenue?.id;
+
   const { recentVenueUsers } = useRecentVenueUsers({ venueName: venue?.name });
 
   const [seatedAtTable, setSeatedAtTable] = useState("");
 
   useExperiences(venue?.name);
 
+  // @debt This logic is a copy paste from NavBar. Move that into a separate Back button component
+  const backToParentVenue = useCallback(() => {
+    if (!parentVenueId) return;
+
+    openUrl(venueInsideUrl(parentVenueId));
+  }, [parentVenueId]);
+
   if (!venue) return <>Loading...</>;
 
```

Comment:
```suggestion
  if (!venue) return <LoadingPage />;

```

Reply:
Actually.. is this even needed anymore? If the `venue` prop can be changed to not have the `?` (which I believe it should be able to), then we can remove this check here entirely.

```suggestion
```

Reply:
fixed in c31874f2e06bf3af652db14f326f77495653a166